Unabel to build xk6-browser

```
go version
go version go1.17.3 darwin/amd64
```

```
sw_vers
ProductName:	macOS
ProductVersion:	11.5.2
BuildVersion:	20G95
```
```
xk6 build --output xk6-browser --with github.com/grafana/xk6-browser@main
2021/11/17 16:49:39 [INFO] Temporary folder: /Users/untitled/buildenv_2021-11-17-1649.864137876
2021/11/17 16:49:39 [INFO] Writing main module: /Users/untitled/buildenv_2021-11-17-1649.864137876/main.go
2021/11/17 16:49:39 [INFO] Initializing Go module
2021/11/17 16:49:39 [INFO] exec (timeout=10s): /usr/local/go/bin/go mod init k6
go: creating new go.mod: module k6
go: to add module requirements and sums:
	go mod tidy
2021/11/17 16:49:39 [INFO] Pinning versions
2021/11/17 16:49:39 [INFO] exec (timeout=0s): /usr/local/go/bin/go mod edit -require go.k6.io/k6@latest
2021/11/17 16:49:39 [INFO] exec (timeout=0s): /usr/local/go/bin/go mod tidy
go: finding module for package github.com/grafana/xk6-browser
go: found github.com/grafana/xk6-browser in github.com/grafana/xk6-browser v0.1.0
2021/11/17 16:49:40 [INFO] exec (timeout=0s): /usr/local/go/bin/go mod edit -require github.com/grafana/xk6-browser@main
2021/11/17 16:49:40 [INFO] exec (timeout=0s): /usr/local/go/bin/go mod tidy
2021/11/17 16:49:41 [INFO] Build environment ready
2021/11/17 16:49:41 [INFO] Building k6
2021/11/17 16:49:41 [INFO] exec (timeout=0s): /usr/local/go/bin/go mod tidy
2021/11/17 16:49:41 [INFO] exec (timeout=0s): /usr/local/go/bin/go build -o /Users/untitled/xk6-browser -ldflags -w -s -trimpath
# github.com/grafana/xk6-browser
../go/pkg/mod/github.com/grafana/xk6-browser@v0.1.1-0.20211116085927-baaf58caef73/main.go:43:3: undefined: modules.InstanceCore
../go/pkg/mod/github.com/grafana/xk6-browser@v0.1.1-0.20211116085927-baaf58caef73/main.go:50:3: undefined: modules.InstanceCore
../go/pkg/mod/github.com/grafana/xk6-browser@v0.1.1-0.20211116085927-baaf58caef73/main.go:56:4: undefined: modules.IsModuleV2
../go/pkg/mod/github.com/grafana/xk6-browser@v0.1.1-0.20211116085927-baaf58caef73/main.go:67:40: undefined: modules.InstanceCore
2021/11/17 16:49:42 [INFO] Cleaning up temporary folder: /Users/untitled/buildenv_2021-11-17-1649.864137876
2021/11/17 16:49:42 [FATAL] exit status 2
````

For example xk6-notification has been built fine

--- a/go.mod
+++ b/go.mod
@@ -10,10 +10,9 @@ require (
 	github.com/mailru/easyjson v0.7.7
 	github.com/mccutchen/go-httpbin v1.1.2-0.20190116014521-c5cb2f4802fa
 	github.com/oxtoacart/bpool v0.0.0-20190530202638-03653db5a59c
-	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.8.1
 	github.com/stretchr/testify v1.7.0
-	go.k6.io/k6 v0.34.2-0.20211105090803-dae70006ca77
+	go.k6.io/k6 v0.35.0
 	golang.org/x/net v0.0.0-20211104170005-ce137452f963
 	gopkg.in/guregu/null.v3 v3.5.0
 )

--- a/main.go
+++ b/main.go
@@ -40,21 +40,20 @@ type (
 
 	// JSModule is the entrypoint into the browser JS module
 	JSModule struct {
-		k6modules.InstanceCore
+		vu      k6modules.VU
 		Devices map[string]common.Device
 		Version string
 	}
 
 	// ModuleInstance represents an instance of the JS module.
 	ModuleInstance struct {
-		k6modules.InstanceCore
 		mod *JSModule
 	}
 )
 
 var (
-	_ k6modules.IsModuleV2 = &RootModule{}
-	_ k6modules.Instance   = &ModuleInstance{}
+	_ k6modules.Module   = &RootModule{}
+	_ k6modules.Instance = &ModuleInstance{}
 )
 
 // New returns a pointer to a new RootModule instance.
@@ -62,22 +61,21 @@ func New() *RootModule {
 	return &RootModule{}
 }
 
-// NewModuleInstance implements the modules.IsModuleV2 interface to return
+// NewModuleInstance implements the k6modules.Module interface to return
 // a new instance for each VU.
-func (*RootModule) NewModuleInstance(m k6modules.InstanceCore) k6modules.Instance {
+func (*RootModule) NewModuleInstance(vu k6modules.VU) k6modules.Instance {
 	return &ModuleInstance{
-		InstanceCore: m,
 		mod: &JSModule{
-			InstanceCore: m,
-			Devices:      common.GetDevices(),
-			Version:      version,
+			vu:      vu,
+			Devices: common.GetDevices(),
+			Version: version,
 		},
 	}
 }
 
-// GetExports returns the exports of the JS module so that it can be used in
-// test scripts.
-func (mi *ModuleInstance) GetExports() k6modules.Exports {
+// Exports returns the exports of the JS module so that it can be used in test
+// scripts.
+func (mi *ModuleInstance) Exports() k6modules.Exports {
 	return k6modules.Exports{Default: mi.mod}
 }
 
@@ -93,11 +91,11 @@ func (m *JSModule) Launch(browserName string, opts goja.Value) api.Browser {
 	}()*/
 
 	if browserName == "chromium" {
-		bt := chromium.NewBrowserType(m.GetContext())
+		bt := chromium.NewBrowserType(m.vu.Context())
 		return bt.Launch(opts)
 	}
 
-	k6common.Throw(m.GetRuntime(),
+	k6common.Throw(m.vu.Runtime(),
 		errors.New("Currently 'chromium' is the only supported browser"))
 	return nil
 }

